docs/getting_started.md

```diff
@@ -11,11 +11,24 @@ weight=1
 
 # Getting started with Docker Notary
 
-This document describes basic use of Notary as a tool supporting content trust.
-For more advanced use cases, you must [run your own Notary
+This document describes basic use of the Notary CLI as a tool supporting Docker
+Content Trust. For more advanced use cases, you must [run your own Notary
 service](running_a_service.md) and should read the [use the Notary client for
 advanced users](advanced_usage.md) documentation.
 
+## What is Notary
+
+Notary is a tool for publishing and managing trusted collections of content.
+Publishers can digitally sign collections and consumers can verify integrity
+and origin of content. This ability is built on a straightforward key management
+and signing interface to create signed collections and configure trusted publishers.
+
+With Notary anyone can provide trust over arbitrary collections of data. Using
+[The Update Framework (TUF)](http://theupdateframework.com/) as the underlying
```

Comment:
I'm not really sure - their spec seems to link to [http://www.theupdateframework.com](https://github.com/theupdateframework/tuf/blob/1bed3e09a478c2c918ffbff10b9118f6e52ee129/docs/tuf-spec.txt#L975).  When I searched for "theupdateframework.github.io" in the repo, I didn't see anything - it mostly seems to link to the github project.  I'll go with "https://www.theupdateframework.com", thanks :)